src/components/App/App.js

```diff
@@ -28,8 +29,69 @@ class App extends Component {
 
   componentDidMount() {
     // fetch will go here I think and assign the actual array to this.state.movies
+    
+    fetch('https://swapi.co/api/films/')
+      .then(response => response.json())
+      .then(data => this.setState({ movies: data.results }))
+      .catch(err => console.log(err))
   }
 
+  fetchChar = (episode) => {
+    let selectedMov = this.state.movies.find(movie => movie.episode_id === episode)
+
+    let promises = selectedMov.characters.map(character => {
+      return fetch(character)
+        .then(response => response.json())
+        .then(charData => {
+          fetch(charData.homeworld)
+            .then(response => response.json())
+            .then(homeworldData => {
+              // let movChar = {
+              //   name: charData.name,
+              //   homeworld: homeworldData,
+              //   hwPop: homeworld.population,
+              //   species: speciesData,
+              //   films: []
+              // }
+              console.log('HW: ', homeworldData)
+            })
+          fetch(charData.species)
+            .then(response => response.json())
+            .then(speciesData => {
+              // movChar = {
+              //   name: charData.name,
+              //   homeworld: homeworldData,
+              //   hwPop: homeworld.population,
+              //   species: speciesData,
+              //   films: []
+              // }
+              // return movChar;
+              console.log('species: ', speciesData)
+            })
+          charData.films.map(film => {
+            return fetch(film)
+              .then(response => response.json())
+              .then(filmData => {
+              //   movChar.films.push(filmData)
+                console.log('film: ', filmData)
+              })
+          })
+          })
+    })
+    Promise.all(promises)
+      // .then(bios => bios.map(bio => {
+      //   let movChar = {
+      //     name: charData.name,
+      //     homeworld: homeworldData,
+      //     hwPop: homeworld.population,
+      //     species: speciesData,
+      //     films: []
+      //   }
+      //   return movChar
+      // }))
+      // .then(bios => this.setState({ characters: bios }))
+    console.log(this.state.characters)
+  }
```

Comment:
@ckoga  - When my Header branch was merged it created a merge conflict in this file between this fetch-branch and the master.   

To resolve this merge conflict we will want to keep the changes from your fetch-branch and delete the conflict master (look like just a single, blank line)